dao/tx/tx.go

```diff
@@ -61,15 +61,15 @@ type (
 		gorm.Model
 
 		// Assigned when created in the tx pool.
-		TxHash       string `gorm:"uniqueIndex"`
-		TxType       int64
-		TxInfo       string
-		AccountIndex int64
-		Nonce        int64
-		ExpiredAt    int64
+		TxHash string `gorm:"uniqueIndex"`
```

Comment:
There is no change in the Tx struct, Can you keep the code format as before?

Reply:
OK.
I changed this because I think `AccountIndex, Nonce, ExpiredAt` are actually assigned after executed
(According to the `// Assigned when created in the tx pool.` and `// Assigned after executed.` comment block,  I move the fields to the correct position I think)

But I can change this back, thanks 🙏